lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -951,11 +951,13 @@ bool HandshakeBuilder::visitHandshake(MergeOp op) {
 
   // Declare wire for arbitration winner.
   auto winName = rewriter.getStringAttr("win");
-  auto win = rewriter.create<WireOp>(insertLoc, indexType, winName);
+  auto win = rewriter.create<WireOp>(insertLoc, indexType, winName,
+                                     rewriter.getArrayAttr({}));
```

Comment:
Can this be defaulted in the WireOp build method?

Reply:
The default is actually there, but not for the `StringAttr name` build variant (only for the `StringRef name`). I assume this is because creating attributes needs a builder which can't work for a default.

I think the right approach is to change this to: `rewriter.create<WireOp>(insertLoc, indexType, "win")`. That picks up the default and the modifications to handshake are minimal.

Reply:
Full WireOp::build variants:

Reply:
*WireOp already has a nice builder with default. `RegOp` and `RegResetOp` could use the same.

Reply:
Cleaned this up in the rebase. Some prerequisite work done to make this simpler in f34013d1 and cccd9f3b.